ogticrd/covid-safe-paths
# Feat: Decrypt  concern points

#### Description:


Decoding concern points from Contact endpoint and fixing a bug in the calendar in the dashboard 
#### Linked issues:


Feat: [273](https://trello.com/c/6FZOLmw7/273-as-a-user-i-want-to-descript-the-concern-points-so-that-my-phone-can-know-when-i-was-in-contact-with-a-positive-user-3)

#### How to test:


- `git pull && git checkout Feat/273-descript-concern-points`
- `yarn install`
- Head over to `check contact` in home screen
- The check contact should work as expected and the calendar should match coronavirusrd data when you open the app the first time
